src/js/closable-surveys.js

```diff
@@ -0,0 +1,89 @@
+import LocalStorageManager from './localStorageManager';
+
+// eslint-disable-next-line func-names
+(function (Drupal) {
+  Drupal.behaviors.closable_surveys = {
+    attach: function attach() {
+      const survey = document.getElementById('helfi-survey__container');
+      if (!survey) return;
+
+      const root = document.documentElement;
+      const surveyDelay = 2000;
+      const surveyCloseButton = document.getElementById('helfi-survey__close-button');
+      const surveyKey = 'hidden-helfi-surveys';
+      const storageManager = new LocalStorageManager('helfi-settings');
+      let surveysToHide = null;
+      let surveyFocusTrap = null;
+
+      try {
+        surveysToHide = JSON.parse(window.localStorage.getItem('helfi-settings'));
+      } catch (e) {
+        console.error('Error parsing local storage data:', e);
+      }
+
+      function addToSurveyStorage() {
+        const { uuid } = survey.dataset;
+        if (uuid) {
+          storageManager.addValue(surveyKey, uuid);
+        }
+      }
+
+      function addFocusTrap() {
+        surveyFocusTrap = window.focusTrap.createFocusTrap('#helfi-survey', {
+          initialFocus: () => '#helfi-survey__title'
+        });
+        surveyFocusTrap.activate();
```

Comment:
Focus trap (or something else here?) seems to disable cookie consent banner buttons while survey popup is drawn under the banner.
<img width="364" alt="image" src="https://github.com/user-attachments/assets/1d84d53e-1bec-41c6-b13a-d8ee75d2e2f6">